scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Splitting build_funcs apart from this is a small quality of life improvement
+# for when testing these locally. You don't want to source the file to get the
+# functions but also necessarily set these values which will exit the terminal.
```

Comment:
🤔 I never thought of this but it's an interesting workflow. Does this mean that if I have `some_custom_func()` in the `build_funcs.sh` that I `source`d (without `build.sh`), and the 4th out of 6 lines in that function fails, will it still continue? Or will it abort?

Reply:
Yes, if you want `set -e` then you need to set it in your terminal first. This is the best way I can think to do it, otherwise you have no choice about it (since even if you set +e, the script will reset -e and close your terminal on failure). And like I found, exiting the terminal youre working in probably isn't what you wanted. Especially for tests usually when testing locally you just want to run and see pass/fail.

Reply:
heh I wish there is a middle ground of "don't continue the rest of the lines, but also don't quit the current shell"

¯\\_(ツ)\_/¯